portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,19 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Camilo Pareja's Portfolio</h1>
+      <!-- <img src="/images/portrait.jpg" id="portrait" /> -->
```

Comment:
In the future, don't leave commented-out code like this.

Reply:
Ok, thanks for the tip!